sys-devel/flang/flang-13.0.0.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+
+PYTHON_COMPAT=( python3_{8..10} )
+inherit cmake-multilib llvm python-single-r1
+
+DESCRIPTION="Flang is a ground-up implementation of a Fortran front end written in modern C++"
+HOMEPAGE=" https://github.com/llvm/llvm-project/tree/main/flang
+		http://flang.llvm.org/docs/"
+SRC_URI="https://github.com/llvm/llvm-project/releases/download/llvmorg-${PV}/llvm-project-${PV}.src.tar.xz
+	https://src.fedoraproject.org/rpms/flang/raw/rawhide/f/0001-Link-against-libclang-cpp.so.patch -> flang-Link-against-libclang-cpp.so.patch
+	https://src.fedoraproject.org/rpms/flang/raw/rawhide/f/0001-PATCH-flang-Disable-use-of-sphinx_markdown_tables.patch -> flang-PATCH-flang-Disable-use-of-sphinx_markdown_tables.patch
+"
+
+LICENSE="Apache-2.0-with-LLVM-exceptions UoI-NCSA MIT"
+SLOT="13"
+IUSE="doc test +clang"
+KEYWORDS="~amd64"
+
+DEPEND="
+	>=sys-devel/clang-13.0.0:13=[static-analyzer,${MULTILIB_USEDEP}]
+	sys-devel/llvm:13=[${MULTILIB_USEDEP}]
+"
+RDEPEND="${DEPEND}
+	${PYTHON_DEPS}"
+
+BDEPEND="
+	doc? (
+		dev-python/sphinx
+		app-doc/doxygen[dot]
+	)
+	test? (
+		dev-python/lit
+	)
+	sys-devel/mlir:13=
+"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+RESTRICT="!test? ( test )"
+
+LLVM_MAX_SLOT=13
+
+pkg_setup() {
+	use system-llvm && llvm_pkg_setup
+	if use clang ; then
```

Comment:
for use clang, you need to pull in sys-devel/lld. Otherwise this happens: 
```
CMake Error at /usr/lib/llvm/13/lib64/cmake/llvm/HandleLLVMOptions.cmake:298 (message):
  Host compiler does not support '-fuse-ld=lld'
Call Stack (most recent call first):
```